.github/labels.yaml

```diff
@@ -0,0 +1,808 @@
+
+- name: needs-triage
+  kind: label
+  spec:
+    match: triage/*
+    matchCondition: NOT
+  actions:
+  - kind: apply-label
+    spec:
+      label: needs-triage
+
+- name: needs-kind
+  kind: label
+  spec:
+    match: kind/*
+    matchCondition: NOT
+  actions:
+  - kind: apply-label
+    spec:
+      label: needs-kind
+
+- name: needs-group
+  kind: label
+  spec:
+    match: "{toc,tag/*,sub/*}"
+    matchCondition: NOT
+  actions:
+  - kind: apply-label
+    spec:
+      label: "needs-group"
+
+- name: apply-kind
+  kind: match
+  spec:
+    command: /kind
+    rules:
+    - matchList:
+      - needs-kind
+      - kind/dd
+      - kind/docs
+      - kind/enhancement
+      - kind/initiative
+      - kind/meeting
+      - kind/review
+      - kind/subproject
+  actions:
+  - kind: remove-label
+    spec:
+      match: needs-kind
+  - kind: apply-label
+    spec:
+      label: kind/{{ argv.0 }}
+
+- name: apply-triage
+  kind: match
+  spec:
+    command: /triage
+    rules:
+    - matchList:
+      - needs-triage
+      - triage/valid
+      - triage/needs-information
+      - triage/duplicate
+      - triage/not-planned
+  actions:
+  - kind: remove-label
+    spec:
+      match: needs-triage
+  - kind: remove-label
+    spec:
+      match: triage/*
+  - kind: apply-label
+    spec:
+      label: triage/{{ argv.0 }}
+
+
+##############################################################################
+# Group (toc/tag/sub) apply commands
+##############################################################################
+
+- name: apply-tag
+  kind: match
+  spec:
+    command: /tag
+    rules:
+    - matchList:
+      - needs-group
+      - tag/developer-experience
+      - tag/infrastructure
+      - tag/operational-resilience
+      - tag/security-compliance
+      - tag/workloads-foundation
+  actions:
+  - kind: remove-label 
+    spec:
+      match: needs-group
+  - kind: apply-label
+    spec:
+      label: tag/{{ argv.0 }} 
+
+- name: apply-toc
+  kind: match
+  spec:
+    command: /toc
+    rules:
+    - match: "toc"
+  actions:
+  - kind: remove-label
+    spec:
+      match: needs-group
+  - kind: apply-label
+    spec:
+      label: toc
+
+- name: apply-sub
+  kind: match
+  spec:
+    command: /sub
+    rules:
+    - matchList:
+      - needs-group
+      - sub/contributor-strategy-and-advocacy
+      - sub/mentoring
+      - sub/project-reviews
+  actions:
+  - kind: remove-label
+    spec:
+      match: needs-group
+  - kind: apply-label
+    spec:
+      label: sub/{{ argv.0 }}
+
+
+##############################################################################
+# DD commands
+##############################################################################
+
+# triage for DD only includes items that are 'blockers'. Any project that has a 
+# complete app, submitted adopters, and has completed their security assessment
+# is ready to be evaluated and dd/status/ready-for-review
```

Comment:
This needs matched with the TOC triage guidance: https://github.com/cncf/toc/blob/main/operations/dd-toc-guide.md#initial-triageevaluation-prior-to-assignment. the TOC Triage guidance is more comprehensive than these three items.

Reply:
This will likely require a new PR to ensure they match/ the TOC Triage guidance is updated.

Reply:
Apologies my comment was less about what the label should be called, and more around 3 != 29 other items therefore we need to align these in our operation's triage guide. We provide links to how the TOC does things to maintainers and community members, it is IMPERATIVE that our labeling schema is reflecting in those docs so anyone can reason and understand the why.

If we're considering those other items as "incomplete" or "not-enough" that is fine, we just need to ensure its clear in the triage documentation we have and incorporates the right label to affix.  potentially, a project could have all three labels affixed so the key is to ensure during Triage, we complete looking at all aspects to label correctly.